source/docs/software/advanced-controls/controls-glossary.rst

```diff
@@ -81,6 +132,9 @@ Controls Glossary
 
       A :term:`system's <system>` state is often represented by the variable :math:`\mathbf{x}`, a column vector with one entry per :term:`state`.
 
+   statistically robust
+      The property of a data processing algorithm which makes it resilient to a wide variety of inputs. Designing statistically robust algorithms on robots is important because real-world sensor data can often be unpredictable, but unexpected robot behavior is never desirable. See `Robust Statistics <https://en.wikipedia.org/wiki/Robust_statistics>`__ on wikipedia for more info.
```

Comment:
I'd say "resilient to noisy inputs" rather than "resilient to a wide variety of inputs."